### DISCUSSION TITLE
How to publish MAUI blazor inside visuals studio?

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/maui#4794] [Q&A] [answered]
I can't understand how to publish (/use the published stuffs) my app though, is it something for users to install or is it something for windows store to distribute

**Comment by Xyncgas** [comment]
> I can't understand how to publish (/use the published stuffs) my app though, is it something for users to install or is it something for windows store to distribute

So that I can let other people try out my app

**Comment by Xyncgas** [comment, marked as answer]
Anyone wanna take this?

**Reply by Ghevi** [reply]
This shows you how to publish on android but at the end there are also link for ios and windows
https://github.com/dotnet/maui/issues/4377

For android you will find for example the apk in platforms/android/bin/
Is this helpfull?
